src/main/kotlin/no/nav/syfo/syfosyketilfelle/client/SyfosyketilfelleClient.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.syfosyketilfelle.client
+
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.client.sts.StsClient
+import no.nav.syfo.domain.AktorId
+import no.nav.syfo.syfosyketilfelle.COUNT_CALL_SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_FAIL
+import no.nav.syfo.syfosyketilfelle.COUNT_CALL_SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_SUCCESS
+import no.nav.syfo.syfosyketilfelle.domain.KOppfolgingstilfellePerson
+import no.nav.syfo.util.bearerHeader
+import org.slf4j.LoggerFactory
+
+class SyfosyketilfelleClient(
+    baseUrl: String,
+    private val stsClient: StsClient,
+) {
+    private val httpClient = httpClientDefault()
+
+    private val syfosyketilfelleOppfolgingstilfellePersonUrl: String =
+        "$baseUrl$SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_PATH"
+
+    suspend fun oppfolgingstilfellePerson(
+        callId: String,
+        aktorId: AktorId,
+    ): KOppfolgingstilfellePerson {
+        try {
+            val stsToken = stsClient.token()
+            val url = "$syfosyketilfelleOppfolgingstilfellePersonUrl/${aktorId.value}"
+            val response: KOppfolgingstilfellePerson = httpClient.get(url) {
```

Comment:
Tar dette høyde for at responsen er NoContent?

Reply:
Nei, glemte den i farten. Skal fikse det.

Reply:
Nå skal den sende 204 videre om den får 204 fra Syfosyketilfelle. La til test for det.